tests/HtmlMatcherTest.php

```diff
@@ -78,14 +78,12 @@ public function considersValidHtml_WHtmlContainsScriptTagWithHtmlContents()
     public function addsSpecificTextInsideTheSciptTagsInsteadOfItsContents()
     {
         $html = "<div>
-<script type='x-template'>
-	<span></span>
-</script>
+<script type='x-template'><span></span></script>
 </div>";
 
         assertThat($html, is(htmlPiece(havingChild(
             both(withTagName('script'))
-                ->andAlso(havingTextContents(HtmlMatcher::SCRIPT_BODY_REPLACEMENT))))));
+                ->andAlso(havingTextContents("<span><\\/span>"))))));
```

Comment:
What you say confuses me a bit. Shouldn't code style be consistent? Either always align `{` and `}` vertically (which is, as a side note, the style I personally preferred before I started doing MediaWiki professionally). Or always have the opening `{` at the end of the line (which is the MediaWiki standard).

If we can agree on using the MediaWiki code style in this repository this will resolve itself. :-)